src/components/UserProfile/AboutMe.js

```diff
@@ -1,45 +1,68 @@
 import * as React from 'react';
 import { xor } from 'lodash';
-import { css } from 'react-emotion';
 import { compose, withState, withPropsOnChange, withHandlers } from 'recompose';
 import { withRouter } from 'react-router-dom';
 import { injectState } from 'freactal';
 import styled from 'react-emotion';
-
 import { withTheme } from 'emotion-theming';
-import EditableLabel from 'uikit/EditableLabel';
-import ExternalLink from 'uikit/ExternalLink';
-import { Container, EditButton, H2, H3, H4 } from './';
+
+import { Container, EditButton, H2, H3, H4 } from './ui';
 import DeleteButton from 'components/loginButtons/DeleteButton';
 import { trackUserInteraction, TRACKING_EVENTS } from 'services/analyticsTracking';
 import InterestsAutocomplete from './InterestsAutocomplete';
 import { ModalActionButton } from '../Modal';
 
+import { Box, Flex, Section } from 'uikit/Core';
+import Row from 'uikit/Row';
+import Column from 'uikit/Column';
+import { HollowButton } from 'uikit/Button';
+import EditableLabel from 'uikit/EditableLabel';
+import ExternalLink from 'uikit/ExternalLink';
+import { Tag } from 'uikit/Tags';
+
 const trackProfileInteraction = ({ action, value, type }) =>
   trackUserInteraction({
     category: TRACKING_EVENTS.categories.user.profile,
     action: `${type || ''} Edit: ${value ? `open` : `close`}`,
     label: action,
   });
 
-const StyledSection = withTheme(styled('section')`
-  ${({ theme }) => theme.section};
+const StyledSection = styled(Section)`
   padding: 5px 0;
   margin-top: 25px;
-  & .clickToAdd {
-    font-size: 12px;
-    text-decoration: underline;
-  }
-`);
-const ClickToAdd = ({ theme, children, ...rest }) => (
-  <a className={`clickToAdd`} {...rest}>
-    {children}
-  </a>
-);
+`;
+
+const AboutMeContainer = styled(Container)`
+  ${({ theme }) => theme.row};
+`;
```

Comment:
Maybe we can add `${({row}) => row ? theme.row : ''}` so we can do `<Container row>` in component? seems like a pretty re-usable pattern.